packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -119,18 +120,31 @@ export class Gauge extends PureComponent<Props> {
   }
 
   render() {
-    const { height, width } = this.props;
+    const { height, width, value } = this.props;
 
     return (
       <div
         style={{
```

Comment:
Since talking Emotion, I think we can replace this with `className` as well

Reply:
I expect the `width`/`height` to vary a lot (as you drag the size, etc) -- it it a ok to generate a class for stuff that has a lot of variations?

Reply:
No, think if there is a lot of variations emotion is not great, will generate a ton of css classes

Reply:
You are absolutely right, haven't paid attention to width/height being style params here.